wagtail/utils/utils.py

```diff
@@ -1,4 +1,8 @@
 from collections.abc import Mapping
+from io import BytesIO
+
+from django.core.files import File
+from PIL import Image
```

Comment:
Issue: For image handling, we tend to use `willow` rather than Pillow directly. 

Aside: Given that, is it possible to reuse Wagtail's renditions resizing here? Perhaps using the operations classes directly from `wagtail/images/image_operations.py`?

Reply:
I took this advice and implemented this functionality in the form itself since the image_operations already implemented the reusability the util function was meant to address. I had to create an image_operation class that subclasses the ScaleOperation though because the ScaleOperation performs scaling by dividing the supplied size/scale by 100 and multiplying that value with the width and height of the image and that prevented testability and expected behaviour of output